jspm.config.js

```diff
@@ -56,6 +56,15 @@ SystemJS.config({
           'typings': true
         }
       }
+    },
+    'src/app/+lazyTest': {
```

Comment:
I don't understand the + ... is this a common naming convention for lazily loaded modules?